File: THCS/2022_2023/CanTho_2223/Bai4/TROCHOI.py
# Đọc dữ liệu từ file TROCHOI.INP
with open('TROCHOI.INP', 'r') as file:
    m, n = map(int, file.readline().split())

# Tính số que cần thiết
result = 2 * (m * n) + m + n

# Ghi kết quả ra file TROCHOI.OUT
with open('TROCHOI.OUT', 'w') as file:
    file.write(str(result))

File: THCS/2022_2023/CanTho_2223/Bai4/TROCHOI.cpp
#include <iostream>
#include <fstream>

using namespace std;

int main() {
    long long m, n;
    
    // Đọc dữ liệu từ file TROCHOI.INP
    ifstream inputFile("TROCHOI.INP");
    inputFile >> m >> n;
    inputFile.close();
    
    // Tính số que cần thiết
    long long result = 2 * (m * n) + m + n;
    
    // Ghi kết quả ra file TROCHOI.OUT
    ofstream outputFile("TROCHOI.OUT");
    outputFile << result;
    outputFile.close();
    
    return 0;
}
